shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/ManagedProcessorSyncService.java

```diff
@@ -36,7 +36,7 @@ private void processDelta(List<ProcessorDTO> processorDTOList) {
             if (processorDTO.getOperationType() == OperationType.DELETE) {
                 managedProcessorService.deleteManagedProcessor(processorDTO);
             } else {
-                managedProcessorService.createManagedProcessor(processorDTO);
+                managedProcessorService.createManagedProcessor(processorDTO, "namespace");
```

Comment:
@lucamolteni You need to take the namespace from the `Bridge`. You can use `NamespaceProvider.getNamespaceName(String bridgeId)` and take the `bridgeId` from the `ProcessorDTO`.